src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -777,62 +777,67 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
+                                                if ( & { Set-StrictMode -Version 1; $_.PSMessageDetails } ) {
+                                                    "" : $($_.PSMessageDetails)""
+                                                }
 
-                                        $errorCategoryMsg = & { Set-StrictMode -Version 1; $_.ErrorCategory_Message }
+                                                if ($_.InvocationInfo -and ($_.InvocationInfo.MyCommand -or ($_.CategoryInfo.Category -ne 'ParserError'))) {
+                                                    $_.InvocationInfo.PositionMessage
+                                                }
 
-                                        if ($null -ne $errorCategoryMsg)
-                                        {
-                                            $indentString = ""+ CategoryInfo          : "" + $_.ErrorCategory_Message
-                                        }
-                                        else
-                                        {
-                                            $indentString = ""+ CategoryInfo          : "" + $_.CategoryInfo
-                                        }
+                                                $errorCategoryMsg = & { Set-StrictMode -Version 1; $_.ErrorCategory_Message }
+                                                if ($null -ne $errorCategoryMsg) {
+                                                    ""+ CategoryInfo          : $($_.ErrorCategory_Message)""
+                                                }
+                                                else {
+                                                    ""+ CategoryInfo          : $($_.CategoryInfo)""
+                                                }
 
-                                        $posmsg += ""`n"" + $indentString
+                                                ""+ FullyQualifiedErrorId : $($_.FullyQualifiedErrorId)""
 
-                                        $indentString = ""+ FullyQualifiedErrorId : "" + $_.FullyQualifiedErrorId
-                                        $posmsg += ""`n"" + $indentString
+                                                $originInfo = & { Set-StrictMode -Version 1; $_.OriginInfo }
+                                                if ($originInfo -and $originInfo.PSComputerName) {
+                                                    ""+ PSComputerName        : $($originInfo.PSComputerName)""
+                                                }
+                                            }
+                                        }
 
-                                        $originInfo = & { Set-StrictMode -Version 1; $_.OriginInfo }
+                                        if ($_.Suggestions.Count -gt 0) {
+                                            $Prefix = "" "" * 2
+                                            $NewLine
+                                            ""Suggestions:""
```

Comment:
Keeping the current rendering format, the suggestion should be a different color

Reply:
I don't entirely disagree, but I don't think the suggestion should look like the output. I'm more okay with it looking like it's part of the error than I am with it looking like it's stray output. That might encourage new users to simply drop random text strings to output with their own suggestions rather than investigating how to do it more appropriately, creating an issue of needlessly cluttering the output stream with jagged collections of various object types.

Reply:
I don't think it has to be white like it is today which I agree looks like regular output.  Perhaps a nice green color to make it positive :)

Reply:
Would that be best done with terminal escape sequences?

Reply:
In my opinion, the terminal world (including Windows now) has accepted escape sequences as the standard so I would say yes.

Reply:
Alright, guess I'll go hunting for a suitable green color then!

Reply:
@SteveL-MSFT so I figured out how to do that, and it's nice.

However, given that console colors can be customized... this needs to be configurable, doesn't it? If someone (I mean, I wouldn't ever want to see it, but it could happen) chose a green background for their console, they'd never see suggestions again. Similarly confusing if they elected to change their default foreground color to some shade of green for "that hacker look".

Should there be an environment variable or something for this?

Reply:
@vexx32 I think customization of colors should be discussed here: https://github.com/PowerShell/PowerShell/issues/3611 outside of this PR.  Once we have that solved, we will need to update any code that has raw escape sequences.  In the future, I'd also like to see markdown rendering of strings, so you could just use markdown to add emphasis.  However, none of this exists today.